nautobot/wireless/models.py

```diff
@@ -0,0 +1,257 @@
+from django.core.validators import MinValueValidator
+from django.db import models
+
+from nautobot.core.constants import CHARFIELD_MAX_LENGTH
+from nautobot.core.models import BaseModel
+from nautobot.core.models.fields import JSONArrayField
+from nautobot.core.models.generics import PrimaryModel
+from nautobot.extras.utils import extras_features
+from nautobot.wireless.choices import (
+    RadioFrequencyChoices,
+    RadioStandardChoices,
+    WirelessAuthTypeChoices,
+    WirelessDeploymentModeChoices,
+)
+
+
+@extras_features(
+    "custom_links",
+    "custom_validators",
+    "export_templates",
+    "graphql",
+    "webhooks",
+)
+class AccessPointGroup(PrimaryModel):
+    """
+    An AccessPointGroup is a collection of AccessPoints. It is used to apply common configuration to multiple
+    AccessPoints at once.
```

Comment:
```suggestion
    An AccessPointGroup is a collection of access points. It is used to apply common configuration to multiple
    access points at once.
```
`AccessPoint` makes it seem like that's another data model

Reply:
True, I really wanted to call it Devices, but Architecture wants Access Points.